src/Normalizer/StatementNormalizer.php

```diff
@@ -80,6 +82,16 @@ public function supportsNormalization($data, $format = null)
      */
     public function denormalize($data, $class, $format = null, array $context = array())
     {
+        $version = null;
+
+        if (isset($data['version'])) {
+            $version = $data['version'];
+
+            if (!preg_match('/^1\.0(?:\.\d+)?$/', $version)) {
```

Comment:
Yes. But here, we deserialize Statement that come from the Clients, which means the version can also be in the form of `1.0` according to the [spec](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#version).

> Version MUST be formatted as laid out for the API version header in [Versioning](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#versioning).